eng/illink.targets

```diff
@@ -210,23 +213,15 @@
       <!-- pass the non-embedded descriptors xml file on the command line -->
       <ILLinkArgs Condition="'$(ILLinkDescriptorsLibraryBuildXml)' != ''">$(ILLinkArgs) -x "$(ILLinkDescriptorsLibraryBuildXml)"</ILLinkArgs>
       <ILLinkArgs Condition="'$(ILLinkSubstitutionsLibraryBuildXml)' != ''">$(ILLinkArgs) --substitutions "$(ILLinkSubstitutionsLibraryBuildXml)"</ILLinkArgs>
+      <ILLinkArgs Condition="'@(ILLinkSuppressionsLibraryBuildXml)' != ''">$(ILLinkArgs) --link-attributes &quot;@(ILLinkSuppressionsLibraryBuildXml->'%(FullPath)', '&quot; --link-attributes &quot;')&quot;</ILLinkArgs>
       <!-- suppress warnings with the following codes:
            IL2008: Could not find type A specified in resource B
            IL2009: Could not find method A in type B specified in resource C
-           IL2012: Could not find field A in type B specified in resource C
            IL2025: Duplicate preserve of A in B
-           IL2026: Calling A which has B can break functionality when trimming application code. The target method might be removed.
            IL2035: Unresolved assembly A in DynamicDependencyAttribute on B
-           IL2050: P/invoke method A declares a parameter with COM marshalling. Correctness of COM interop
-                   cannot be guaranteed after trimming. Interfaces and interface members might be removed.
       -->
-      <LinkerNoWarn>IL2008;IL2009;IL2012;IL2025;IL2026;IL2035;IL2050</LinkerNoWarn>
-      <!-- IL2032,IL2055,IL2057-IL2061: Reflection intrinsics with unknown arguments -->
-      <LinkerNoWarn>$(LinkerNoWarn);IL2032;IL2055;IL2057;IL2058;IL2059;IL2060;IL2061</LinkerNoWarn>
-      <!-- IL2062-IL2066: Unknown values passed to locations with DynamicallyAccessedMemberTypes -->
-      <LinkerNoWarn>$(LinkerNoWarn);IL2062;IL2063;IL2064;IL2065;IL2066</LinkerNoWarn>
-      <!-- IL2067-IL2091: Unsatisfied DynamicallyAccessedMembers requirements -->
-      <LinkerNoWarn>$(LinkerNoWarn);IL2067;IL2068;IL2069;IL2070;IL2071;IL2072;IL2073;IL2074;IL2075;IL2076;IL2077;IL2078;IL2079;IL2080;IL2081;IL2082;IL2083;IL2084;IL2085;IL2086;IL2087;IL2088;IL2089;IL2090;IL2091</LinkerNoWarn>
+      <LinkerNoWarn>$(LinkerNoWarn);IL2008;IL2009;IL2035</LinkerNoWarn>
+      <LinkerNoWarn>$(LinkerNoWarn);IL2025</LinkerNoWarn>
       <!-- IL2121: Unnecessary UnconditionalSuppressMessage attribute -->
       <LinkerNoWarn>$(LinkerNoWarn);IL2121</LinkerNoWarn>
```

Comment:
Is suppressing IL2121 still necessary? Where are the unnecessary UnconditionalSuppressMessage attributes? And why are they still needed?